2023/1002xxx/GSD-2023-1002425.json

```diff
@@ -10,8 +10,8 @@
     },
     "osvSchema": {
       "id": "GSD-2023-1002425",
-      "modified": "2023-03-27T16:49:19.714305Z",
-      "published": "2023-03-27T16:49:19.714305Z",
+      "modified": "2023-03-29T10:34:26.905Z",
+      "published": "2023-03-27T16:49:19.714Z",
```

Comment:
Looks like this is a consequence of JavaScript being fussy about the millis...Not sure if it's worth worrying about though, as stuff will all eventually be the full ISO8601 string.

Reply:
Oh, actually on closer look, what happened is JavaScript truncated the string to the final 3 digits, probably because of the mask being used for the ISO8601 string in the datetime input